packages/replayio/src/index.ts

```diff
@@ -12,3 +13,13 @@ import "./commands/upload-source-maps";
 import "./commands/view";
 
 finalizeCommander();
+
+// If the process is terminated by CTRL+C while waiting for an async function
+// avoid ERR_UNHANDLED_REJECTION from being printed to the console
+process.on("uncaughtException", async error => {
+  if (error.name !== "UnhandledPromiseRejection") {
+    console.error("uncaughtException:", error);
+  }
+
+  await exitProcess(1);
+});
```

Comment:
isn't the fact that we get `UnhandledPromiseRejection` a sign that some promise is truly dangling unhandled somewhere?

Reply:
We talked about this above, but it's _expected_ in cases like prompt/confirmation or select-a-list-of-recordings.